projects/example-app/src/app/auth/actions/index.ts

```diff
@@ -1,5 +1,5 @@
-import * as AuthActions from './auth.actions';
-import * as AuthApiActions from './auth-api.actions';
-import * as LoginPageActions from './login-page.actions';
+import { authActions as AuthActions } from './auth.actions';
+import { authApiActions as AuthApiActions } from './auth-api.actions';
+import { loginPageActions as LoginPageActions } from './login-page.actions';
 
 export { AuthActions, AuthApiActions, LoginPageActions };
```

Comment:
Let's remove barrel files for actions. With `createActionGroup`, they are not needed anymore.